library/src/main/java/com/novoda/downloadmanager/lib/DownloadsDestination.java

```diff
@@ -0,0 +1,63 @@
+package com.novoda.downloadmanager.lib;
+
+final class DownloadsDestination {
```

Comment:
the really big file is not something we should shy away from it's a fact that this is the beast of our database and i think it should be shown like that (highlighting the smell potentially)